phpcs.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
```

Comment:
This should be named `.phpcs.xml.dist` or `phpcs.xml.dist`.

Reply:
I based this on the work done at https://github.com/Automattic/amp-wp/ cc @westonruter

Reply:
@westonruter should know better :-)

I've created https://github.com/Automattic/amp-wp/issues/1324 for them to update it at AMP.

Reply:
Renamed with 5767fb0.